app/controllers/api/work_times_controller.rb

```diff
@@ -115,13 +118,34 @@ def find_work_time
     end
 
     def work_time_create_params
-      params.fetch(:work_time).permit(:project_id, :user_id, :body, :task, :starts_at, :ends_at)
+      params.fetch(:work_time)
+            .permit(
+              %i[
+                project_id
+                user_id
+                body
+                task
+                tag
+                starts_at
+                ends_at
+              ]
+            )
     end
 
     def work_time_params
-      params.fetch(:work_time).permit(:project_id, :body, :task, :starts_at, :ends_at)
+      params.fetch(:work_time)
```

Comment:
Why not `require`?

Reply:
I just extended the permit part, didn't even noticed fetch. Ofc should be required